### ISSUE TITLE
UI Design

### ISSUE THREAD

**Issue by naheem3410** [BuildForSDGCohort2/EMRs#2]
Make a general UI design for the project

**Comment by naz-coder** [comment]
I guess I am to work with those features we mentioned on slack. Is there a targeted date for me to have commpleted the UI?
    On Wednesday, August 26, 2020, 06:29:05 AM CDT, naheem3410 <notifications@github.com> wrote:  
 
 


Make a general UI design for the project

**Comment by naheem3410** [comment]
Yes, you are to work with those features. The deadline for the task is this
Friday(Aug 27) morning. Thank you.

**Comment by naheem3410** [comment]
Pardon me, I mean to say the deadline for the task is this Friday(Aug 28)
morning. Thank you
